common/lib/xmodule/xmodule/modulestore/tests/test_publish.py

```diff
@@ -139,3 +157,1222 @@ def test_publish_draft_delete(self):
             self.draft_mongo.get_item(location)
         self.assertNotIn(other_child_loc, item.children)
         self.assertTrue(self.draft_mongo.has_item(other_child_loc), "Oops, lost moved item")
+
+
+class UniversalTestSetup(unittest.TestCase):
```

Comment:
Nit: I think `Universal` is less clear in this context than in `test_field_api.py`, because this is only actually used as the testbase for one type of tests. (It's not talking about universal properties, which is what the `test_field_api.py` version was trying to capture).